src/aks-preview/azext_aks_preview/custom.py

```diff
@@ -43,6 +43,8 @@
 import colorama  # pylint: disable=import-error
 from tabulate import tabulate  # pylint: disable=import-error
 from azure.cli.core.api import get_config_dir
+from azure.cli.core.azclierror import (ArgumentUsageError,
+                                       MutuallyExclusiveArgumentError)
```

Comment:
The minimum required CLI version for `aks-preview` extension is `2.0.49`. To be compatible with older CLI versions, you still need to use `CLIError` in the extension unless you bump the `azext.minCliCoreVersion` to `2.15.0+`. See more in the [doc](https://github.com/Azure/azure-cli/blob/dev/doc/error_handling_guidelines.md#azure-cli-error-handling-guidelines).